crates/astria-core/src/protocol/transaction/v1/action/mod.rs

```diff
@@ -689,18 +749,15 @@ enum ValidatorUpdateErrorKind {
     Secp256k1NotSupported,
     #[error("bytes stored in the .pub_key field could not be read as an ed25519 verification key")]
     VerificationKey { source: crate::crypto::Error },
-    #[error(
-        "validator name was {length} characters long, but must be {MAX_VALIDATOR_NAME_LENGTH} at \
-         most"
-    )]
-    NameTooLong { length: usize },
+    #[error(transparent)]
```

Comment:
This is validating an actual protobuf, right? This should not be `transparent` then. Actually provide the field that you had to parse and that was bad. I.e.:

```suggestion
    #[error("field `.name` was invalid")]
```

Reply:
Done in 336459b